docker-compose.yml

```diff
@@ -26,28 +26,40 @@ services:
         networks:
             - localdev
     npm:
-        image: postgres-django
+        image: hedera-django
         build:
             context: .
             dockerfile: Dockerfile
+            target: prod
         container_name: hedera-npm
-        volumes:
-            - ./:/app
```

Comment:
This container doesn't appear to need anything from the host once it's built. I suppose since it runs `npm install` on each `up` we could include `package.json` and/or `package-lock.json`. Not sure what people prefer here, but I don't think it should include `node_modules`, nor does it need access to python source code.